sktime/transformations/series/tests/test_adi_cv.py

```diff
@@ -9,124 +9,56 @@
 from sktime.transformations.series.adi_cv import ADICVTransformer
 
 
-def _generate_smooth_series(size: int = 750):
```

Comment:
I would leave both tests in, and the generate functions, imo they are useful if we need mor egeneral tests later on.
Perhaps allow the `_generate` functions to set a random seed, or there is a stochatsic failure chance.

Reply:
I agree, perhaps we can specify the `adi_trim_handling` parameter's value to keep the old mathematical formula?